tests/phpunit/tests/admin/exportWp.php

```diff
@@ -323,4 +323,267 @@ public function test_export_wp_includes_comments_when_not_filtered() {
+
+		// Post should be present in export.
+		$found_post = false;
+		foreach ( $xml->channel->item as $item ) {
+			$wp_item = $item->children( 'wp', true );
+			if ( (int) $wp_item->post_id === $post_id ) {
+				$found_post = true;
+				break;
+			}
+		}
+
+		$this->assertTrue( $found_post, 'Post with NULL metadata should be included in export' );
+	}
+
+	/**
+	 * Tests that export handles comments with NULL values without fatal errors.
+	 *
+	 * @ticket 64347
+	 */
+	public function test_export_with_null_comment_values() {
+		global $wpdb;
+
+		$post_id = self::factory()->post->create(
+			array(
+				'post_title' => 'Test Post for Comments',
+				'post_type'  => 'post',
+			)
+		);
+
+		$comment_id = self::factory()->comment->create(
+			array(
+				'comment_post_ID' => $post_id,
+				'comment_content' => 'Test comment',
+			)
+		);
+
+		// Insert NULL comment meta.
+		$wpdb->insert(
+			$wpdb->commentmeta,
+			array(
+				'comment_id' => $comment_id,
+				'meta_key'   => 'null_comment_meta',
+				'meta_value' => null,
+			),
+			array( '%d', '%s', '%s' )
+		);
+
+		$xml = $this->get_the_export(
+			array(
+				'content' => 'post',
+			)
+		);
+
+		$this->assertNotFalse( $xml, 'Export should not fail with NULL comment meta values' );
+		$this->assertGreaterThan( 0, count( $xml->channel->item ), 'Export should contain items' );
+	}
+
+	/**
+	 * Tests that export handles term meta with NULL values without fatal errors.
+	 *
+	 * @ticket 64347
+	 */
+	public function test_export_with_null_term_meta_values() {
+		global $wpdb;
+
+		// Create term.
+		$term = self::factory()->term->create(
+			array(
+				'taxonomy' => 'category',
+				'name'     => 'Test Category',
+			)
+		);
+
+		$post_id = self::factory()->post->create(
+			array(
+				'post_title'  => 'Test Post with Category',
+				'post_type'   => 'post',
+				'post_status' => 'publish',
+			)
+		);
+
+		wp_set_object_terms( $post_id, $term, 'category' );
+
+		// Insert NULL term meta.
+		$wpdb->insert(
+			$wpdb->termmeta,
+			array(
+				'term_id'    => $term,
+				'meta_key'   => 'null_term_meta',
+				'meta_value' => null,
+			),
+			array( '%d', '%s', '%s' )
+		);
+
+		$xml = $this->get_the_export(
+			array(
+				'content' => 'all',
+			)
+		);
+
+		$this->assertNotFalse( $xml, 'Export should not fail with NULL term meta values' );
+		$this->assertGreaterThan( 0, count( $xml->channel->item ), 'Export should contain items' );
+	}
+
+	/**
+	 * Tests that export handles posts with NULL title and content without fatal errors.
+	 *
+	 * @ticket 64347
+	 */
+	public function test_export_with_null_post_fields() {
+		global $wpdb;
+
+		// Create a post first.
+		$post_id = self::factory()->post->create(
+			array(
+				'post_title'   => 'Test Post',
+				'post_content' => 'Test content',
+				'post_type'    => 'post',
+			)
+		);
+
+		// Update to set NULL values directly (bypassing WordPress API).
+		$wpdb->update(
+			$wpdb->posts,
+			array(
+				'post_excerpt' => null,
```

Comment:
The `post_excerpt` field actually cannot be `null`:

https://github.com/WordPress/wordpress-develop/blob/63e679c62b1e6b155b58c8b2bd87e49f3c0c9c98/src/wp-admin/includes/schema.php#L166

Therefore, this test doesn't test a `null` value here. I'll remove this test.

Reply:
Well, I tried running these tests against `trunk` without the fix applied, and this test didn't error, which means that `null` wasn't passed to `wxr_cdata()`.